src/hooks/useMellowLPVaults.ts

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useMemo } from 'react';
+import { providers } from 'ethers';
+import { AugmentedMellowLpVault } from '@utilities';
+
+export type useMellowLPVaultsResult = {
+  lpVaults?: AugmentedMellowLpVault[];
+  loading: boolean;
+  error: boolean;
+};
+
+const useMellowLPVaults = (): useMellowLPVaultsResult => {
+  const handleRefetch = useCallback(async () => {}, []);
+  
+  const lpVaults = useMemo(() => {
+    const addresses = (process.env.REACT_APP_MELLOW_VAULTS) ? 
+    process.env.REACT_APP_MELLOW_VAULTS
+      .replace("(", "")
+      .replace("[", "")
+      .replace(")", "")
+      .replace("]", "")
+      .split(",")
+    : [];
```

Comment:
can you please move this to some sort of utility that you will document and potentially add tests around it. Would be good to understand what type of input REACT_APP_MELLOW_VAULTS can be. Right now it seems as "woodoo magic"

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/commit/a44c3c48e2a2660e1fc14f6eeeda646152d2edea